docs/sample-component/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "private": true,
+  "name": "@hig/sample-component-breadcrumbs",
+  "version": "1.0.0-alpha",
+  "description": "hig sample component with breadcrumbs style",
+  "author": "Autodesk Inc.",
+  "homepage": "https://hig.autodesk.com/packages/breadcrumbs",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/garyps/hig.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/index.js",
+  "module": "build/index.es.js",
+  "style": "build/index.css",
```

Comment:
I would remove this, as we will probably make all components themable with emotion and would not be building out stylesheets anymore.

Reply:
Sure, i have removed this line and updated(rebase) development branch on this branch